modules/yandexBidAdapter.js

```diff
@@ -493,4 +481,11 @@ function addRTT(url, rtt) {
   return url;
 }
 
+/** @returns {{webdriver: boolean}} */
+function getDeviceExt() {
+  return {
+    webdriver: self.navigator?.webdriver === true
```

Comment:
please add this to core, not just your adaper. perhaps, device.ext.webdriver

Reply:
please only populate if true in core, you're selcome to make it look however you want on your payload

Reply:
perhaps https://github.com/prebid/Prebid.js/blob/ff58535df6f60c2289e73ea4b832aef8afed046e/src/fpd/enrichment.js#L112

Reply:
Will do, thanks for the suggestion!